src/esp/core/configure.h.cmake

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#cmakedefine BUILD_EGL_SUPPORT
+
+#cmakedefine BUILD_PTEX_SUPPORT
+
+#cmakedefine BUILD_GLOG_SHIM
```

Comment:
Wait but now it all lost the `ESP_` prefix, especially the `BUILD_EGL_SUPPORT` is quite likely to conflict with something else ... and there's `ESP_USE_GLOG_SHIM` but this is `BUILD_GLOG_SHIM`.

Can it be `BUILD_*` on CMake side and `ESP_BUILD_*` on C++ side? :)

Reply:
Yes :)